ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -283,7 +283,7 @@ private QueryMetadata buildPlanForStructuredOutputNode(
     return new PersistentQueryMetadata(
         sqlExpression,
         streams,
-        outputNode.getSchema(),
+        sinkSchema,
```

Comment:
This ensures we've built the `PersistentQueryMetadata` with a schema that has the ROW_KEY and ROW_TIME fields in the schema. (which is the existing functionality).